apollo-router-core/src/plugin.rs

```diff
@@ -221,3 +233,53 @@ macro_rules! register_plugin {
         }
     };
 }
+
+#[derive(Clone)]
+pub struct Handler {
+    service: Buffer<
+        BoxService<http_compat::Request<Bytes>, http_compat::Response<ResponseBody>, BoxError>,
+        http_compat::Request<Bytes>,
+    >,
+}
+
+impl Handler {
+    pub fn new(
+        service: BoxService<
+            http_compat::Request<Bytes>,
+            http_compat::Response<ResponseBody>,
+            BoxError,
+        >,
+    ) -> Self {
+        Self {
+            service: Buffer::new(service, 20_000),
```

Comment:
At some point we need to define a global static for this somewhere! :)

Reply:
Agree !!!